josemak25/baby-bliss
# Ft post component

#### What does this PR do?

Have an overview feature of the app displayed in get started screen.

#### Description of Task to be completed?

Have the get started screen show the following:

1. Home page to display the post component .
2. Animate the love button on click.

#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`

javascript
git checkout ft-post-component

git pull origin ft-post-component

yarn install

yarn test

Using an emulator: Launch the app, login in and you should be at the home page.

#### Screenshots (if appropriate)

![Simulator Screen Shot - iPhone 11 - 2020-02-29 at 20 41 15](https://user-images.githubusercontent.com/33870502/75614026-5a526e00-5b34-11ea-846e-103fd631112e.png)
